<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>STI2</class>
 <widget class="QStackedWidget" name="STI2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>940</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1100</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>STI 2: El Retorno</string>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="page">
   <property name="minimumSize">
    <size>
     <width>720</width>
     <height>480</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1100</width>
     <height>720</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>247</x>
      <y>239</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>60</height>
     </size>
    </property>
    <property name="text">
     <string>Miniserie: La vida de Amorós</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>247</x>
      <y>351</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>380</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>Miniserie: Las desgracias de Santi</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>659</x>
      <y>351</y>
      <width>211</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>60</height>
     </size>
    </property>
    <property name="text">
     <string>Miniserie: Jorge, otra vida de sufrimiento</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>659</x>
      <y>239</y>
      <width>214</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>60</height>
     </size>
    </property>
    <property name="text">
     <string>Miniserie: Roberto, una vida de sufrimiento</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>490</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Rapunsen</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>470</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Rapunsen Saga:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_11">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>530</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Rapunsen 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_12">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>530</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Rapunsen 4, el secuestro de Big Smoke</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_13">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>490</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Rapunsen 3, la película</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>461</x>
      <y>79</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>60</height>
     </size>
    </property>
    <property name="text">
     <string>Historia Principal</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="page_2">
   <property name="minimumSize">
    <size>
     <width>640</width>
     <height>480</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1100</width>
     <height>720</height>
    </size>
   </property>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>500</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Atrás</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>210</y>
      <width>461</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Tras derrotar a Torresano, nuestros héroes regresan una vez más al Satafi, esta vez para hacer frente a Jesucristo, el más poderoso de todos los profesores.&lt;/p&gt;&lt;p&gt;En esta nueva entrega se encontrarán con nuevos personajes: Roberto el emprendedor fracasado, el señor de los ventiladores, Abel...&lt;/p&gt;&lt;p&gt;Descubrirán algunos de los oscuros secretos que esconde el Satafi, y el STI llegará a un nivel de caos nunca antes visto.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Historia Principal</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>170</y>
      <width>481</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>------------------------------------------------------------------</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>460</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Listado de coñas</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>611</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/images/STI2_Title.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>160</y>
      <width>351</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/images/STI2_1.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_14">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>460</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Galería de vídeos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_15">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>500</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Galería de audios</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_17">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>500</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Galería de imágenes</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>320</y>
      <width>241</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/images/STI2_2.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>350</y>
      <width>131</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/images/STI2_3.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>360</y>
      <width>91</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/images/STI2_4.JPG</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="page_3">
   <property name="minimumSize">
    <size>
     <width>980</width>
     <height>720</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1100</width>
     <height>720</height>
    </size>
   </property>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>550</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Anterior Capítulo</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDESC">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>461</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string>XXX</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelTITU">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>701</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>XXX</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelCAP">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>71</y>
      <width>481</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>XXX</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_16">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>550</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Siguiente Capítulo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>600</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Volver al Menú</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelIMG">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>92</y>
      <width>491</width>
      <height>431</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>1100</width>
      <height>720</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelIMG2">
    <property name="geometry">
     <rect>
      <x>76</x>
      <y>302</y>
      <width>441</width>
      <height>231</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>720</width>
      <height>480</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelIMG2DESC">
    <property name="geometry">
     <rect>
      <x>76</x>
      <y>473</y>
      <width>421</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelIMGDESC">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>350</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
